src/components/tableOfContents/tableOfContents.js

```diff
@@ -0,0 +1,69 @@
+//sidebar left is used in layout.js
+import React from "react";
+import "./tableOfContents.css";
+
+/*export default function TableOfContents(props){
+	return(
+		<nav id="table-of-contents" dangerouslySetInnerHTML={{ __html: props.tableOfContentsHtml }}/>
+	)
+}//*/
+
+//https://stackoverflow.com/questions/9979827/change-active-menu-item-on-page-scroll
+
+export default class TableOfContents extends React.Component{
+	//constructor(props){
+		//super(props)
+		//this.state = {someVar : someVal	}
+	//}
+
+	_getPageYOffset = ()=>{	// cross-browser fn to get pageYOffset
+		//if(typeof window === 'undefined')return;
+		if(window.pageYOffset) return window.pageYOffset; // Firefox, Chrome, Opera, Safari.
+		if(document.documentElement && document.documentElement.scrollTop) return document.documentElement.scrollTop; // Internet Explorer 6 (standards mode).
+		if(document.body.scrollTop) return document.body.scrollTop; // Internet Explorer 6, 7 and 8.
```

Comment:
do not add code for anything below ie10. It does not make sense for us to support so much old browsers.

Reply:
Resolved with [3rd commit](https://github.com/fabricjs/fabricjs.github.io/pull/28/commits/a5a0d2c79af191d786ae36fa1b876cf2a71a9af8) 😊